src/components/Form.js

```diff
@@ -17,10 +17,10 @@ export default function Form({
     if (editReviewState) {
       editReview({ ...newReview, id: editReviewState.id })
       setEditReviewState(undefined)
-      history.push('/')
+      history.push('/reviews')
```

Comment:
You could put this line outside the if statement, since it will run in any case.

Reply:
Good call, I will change this